app/bundles/CoreBundle/Tests/Unit/Templating/Twig/Extension/AssetExtensionTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CoreBundle\Tests\Unit\Templating\Twig\Extension;
+
+use Mautic\CoreBundle\Helper\PathsHelper;
+use Mautic\CoreBundle\Templating\Helper\AssetsHelper;
+use Mautic\CoreBundle\Templating\Twig\Extension\AssetExtension;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Asset\Packages;
+
+class AssetExtensionTest extends TestCase
+{
+    /**
+     * @return MockObject|Packages
+     */
+    private function createPackagesMock()
+    {
+        $packagesMock = $this->getMockBuilder(Packages::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $packagesMock->method('getUrl')
+            ->will($this->returnCallback(function (string $path) {
+                return $path;
+            }));
+
+        return $packagesMock;
+    }
+
+    public function testGetCountryFlag(): void
+    {
+        $pathsHelper = $this->getMockBuilder(PathsHelper::class)
+             ->disableOriginalConstructor()
+             ->getMock();
+        $pathsHelper->method('getSystemPath')->willReturn('');
+
+        $assetHelper = new AssetsHelper($this->createPackagesMock());
```

Comment:
The test could be written with shorter syntax if you'd insist on a unit test:
```suggestion
    public function testGetCountryFlag(): void
    {
        $packagesMock = $this->createMock(Packages::class);
        $packagesMock->method('getUrl')
            ->willReturnCallback(fn (string $path) => $path));

        $pathsHelper = $this->createMock(PathsHelper::class);
        $pathsHelper->method('getSystemPath')->willReturn('');

        $assetHelper = new AssetsHelper($this->createPackagesMock());
```

but it might be more valuable and even simpler to call the live service as I'm not quite sure what this test is testing. Something like:

```php
class AssetExtensionTest extends AbstractMauticTestCase
{
    public function testGetCountryFlag(): void
    {
        $assetExtension = self::container->get(AssetExtension::class);
        \assert($assetExtension instanceof AssetExtension);

        Assert::assertSame('whatever the real data should have been returned here', $extension->getCountryFlag('US'));
    }
}
```

Reply:
I did the change in https://github.com/mautic/mautic/pull/11815/commits/e9552678e7954c6b3d02234cd8ef8e502773687e so I could merge this. Hope you don't mind.